packages/flutter_tools/test/web/devices_test.dart

```diff
@@ -26,6 +26,20 @@ void main() {
         return null;
       });
     });
+
+    test('Defaults', () async {
+      final WebDevice device = WebDevice();
+
+      expect(device.name, 'Chrome');
+      expect(device.id, 'web');
```

Comment:
I agree w/ @DanTup's comment - `chrome` seems like a good ID to use here.

Reply:
Renamed to chrome